Lawnicons as icon pack (core changes) (#815)
LawnchairLauncher/lawnicons: CONTRIBUTING.md

## Adding an icon to Lawnicons
Here’s how to add an icon to&nbsp;Lawnicons:

1. Prepare your icon in the SVG format, adhering to the [above guidelines](#icon-guidelines). Use snake case for the filename (e.g.,&nbsp;`youtube_music.svg`).

1. Add the ready SVG to the `svgs`&nbsp;directory.

1. Add a new line to `app/assets/appfilter.xml` (in alphabetical order), and map the new icon to a package name and app's activity. For&nbsp;example:

    ```xml

>     <item component="ComponentInfo{app.lawnchair.lawnicons/app.lawnchair.lawnicons.MainActivity}" drawable="lawnicons"/> 

Reviewer: I'd recommend keeping the same app/package name as before (Files by Google) for consistency with the rest of the steps/images.

This also includes the bottom section.

    ```

    A general template is as&nbsp;follows:

    ```xml
    <item component="ComponentInfo{[PACKAGE_NAME]/[APP_ACIVITY_NAME]}" drawable="[DRAWABLE NAME]"/> 
    ```
1. Done! You’re ready to open a pull request. Please set `develop` as the base&nbsp;branch.

> ## How to find App Package and App Activity of your Android App

Reviewer: ```suggestion
## How to find the package and activity name of your app
```

> ### Using adb via computer/phone with OTG.

Reviewer: The periods are unnesessary in the headers.

>   1. Connect your Android device or emulator to the PC/Mac/Phone via otg and open the App whose details you want to inspect i.e Whatsapp.
>   1. Open a Command Prompt or Terminal window and use ‘adb devices‘
>   1. Finally, type the below-given command to get the information about the currently open application.

Reviewer: ```suggestion
  1. Connect your Android device or emulator to the PC/Mac/Phone via otg and open the app whose details you want to inspect, i.e. Files by Google.
  1. Open a new Command Prompt or Terminal window and input `adb devices`
  1. Finally, type the below-given command to get the information about the currently open application.
```

>   #### For Mac/Linux:

Reviewer: I think bolding this text instead of making it a subheader makes more sense:

```suggestion
  **For Mac/Linux**:
```

  ```xml
  adb shell dumpsys window | grep -E 'mCurrentFocus'  
  ```

>   #### For Windows:

Reviewer: ```suggestion
  **For Windows**:
```

  ```xml
  adb shell dumpsys window | find "mCurrentFocus"
  ```
  ![](./contributing-image-5.png)

>   Here, the part before the '/' character i.e _org.telegram.messenger_ is the package name [PACKAGE_NAME] and the part after that i.e _org.telegram.messenger_.DefaultIcon is the Activity name [APP_ACIVITY_NAME].

Reviewer: ```suggestion
  Here, the part before the `/` character i.e `org.telegram.messenger` is the package name `[PACKAGE_NAME]` and the part after that i.e `org.telegram.messenger_.DefaultIcon` is the Activity name `[APP_ACIVITY_NAME]`.
```

  Here, the part before the '/' character i.e _org.telegram.messenger_ is the package name [PACKAGE_NAME] and the part after that i.e _org.telegram.messenger_.DefaultIcon is the Activity name [APP_ACIVITY_NAME].

### Using 3rd Party Apps.

>   1. Download [Numix Icon Request] (https://f-droid.org/is/packages/org.numixproject.iconsubmit/). 

Reviewer 2: i would use this app for icon requests
https://github.com/Kaiserdragon2/IconRequest/releases

Author: @daywalk3r666 yes it's better let me update it,
